packages/generator/test/prisma/field.test.ts

```diff
@@ -62,21 +60,21 @@ describe("Field model", () => {
 
   it("handles hasOne relations", () => {
```

Comment:
Prevent automatic conversion of field name when creating a Field with relationships, such as `hasOne`.